src/cljs/rems/administration/catalogue_items.cljs

```diff
@@ -131,11 +150,16 @@
                                     :sortable? false
                                     :filterable? false}]
                          :rows [::catalogue-table-rows]
-                         :default-sort-column :name}]
+                         :default-sort-column :name
+                         :selectable? true
+                         :on-select #(rf/dispatch [::set-selected-items! %])}]
     [:div.mt-3
      [table/search catalogue-table]
      [table/table catalogue-table]]))
 
+(defn- items-by-id [items ids]
+  (filter (comp ids :id) items))
```

Comment:
For me items-by-id would be the name for a map of format `{id item}`, but this function is something else. Maybe extract the predicate to a named function and use filter directly.

Reply:
I think it's missing an s.

Reply:
Something like `filter-items-by-ids` or `select-items-with-ids` would make it more obvious what the function does.

Reply:
I have to say that I agree with Esko here. :) With its current name, I was expecting the return value to be a map with ids as keys.